scrapinghub/hubstorage/collectionsrt.py

```diff
@@ -32,7 +32,7 @@ def _allows_mpack(self, path=None):
         :param path: None, tuple or string
 
         """
-        if not MSGPACK_AVAILABLE:
+        if not MSGPACK_AVAILABLE or not self.client.use_msgpack:
```

Comment:
Added a warning message to illustrate what I meant in https://github.com/scrapinghub/python-scrapinghub/pull/73/commits/736e57da91833a3ceb9b6e455c61fa4e795c85d1.